src/ext/conversion/importProperties.ts

```diff
@@ -1,13 +1,49 @@
 import conversion from 'qlik-object-conversion';
+import { setValue } from 'qlik-chart-modules';
 
 import data from '../../qae/data';
-import { ExportFormat, PropTree } from '../../types';
+import { ExportFormat, PropTree, DimensionProperties, MeasureProperties } from '../../types';
+import { ColumnWidthTypes } from '../../table/constants';
+
+export const getColumnInfo = (
+  columnInfo: DimensionProperties[] | MeasureProperties[],
+  colIdx: number,
+  columnWidths: unknown,
+  numDims?: number
+) => {
+  let index = colIdx;
+  if (numDims) index = colIdx - numDims;
+  const column = columnInfo[index];
```

Comment:
if you spread into a new variable here, you can just do column.qDef.columnWidth = { ... } in the if statement

Reply:
and then only have one return at the bottom

Reply:
Fixed